stable/airflow/templates/deployments-flower.yaml

```diff
@@ -112,19 +112,40 @@ spec:
                && source /home/airflow/airflow_env.sh \
                && echo "*** running flower..." \
                && exec airflow flower
+          {{- if .Values.flower.livenessProbe.enabled }}
```

Comment:
Why do we want to support disabling the livenessProbe of flower?

Reply:
Only added it because `deployments-web.yaml` also had `.Values.web.livenessProbe.enabled`, However "web" also accepts all other `livenessProbe` values, so maybe it was no reason for me to only add the `enabled`value without adding the rest. Should I just remove `flower.livenessProbe.enabled` for now, or add all the other values?

Reply:
Those other values are probably not necessary, as we don't allow installing requirements/pip in the flower container. (Which we use `livenessProbe.initialDelaySeconds` to give more time for)

Reply:
Would it be better to just not add an option to disable livenessProbe for flower (and if that is needed it can be its own PR)? I don't want to add an option unless it is going to be useful to people.